openstates/dc/committees.py

```diff
@@ -13,32 +13,25 @@ def scrape(self, term, chambers):
         doc = lxml.html.fromstring(data)
         doc.make_links_absolute(com_url)
 
-        urls = set(doc.xpath('//a[contains(@href, "committee-on")]/@href'))
-        for url in urls:
-            data = self.urlopen(url)
-            doc = lxml.html.fromstring(data)
-
-            try:
-                name = doc.xpath('//h1/text()')[0].replace('Committee on ', '')
-            except IndexError:
-                name = doc.xpath('//h2/text()')[0].replace('Committee on ', '')
-
-
-            # skip link to Committees page
-            if name == 'Committees':
-                continue
-
-            com = Committee('upper', name)
-
-            chairs = []
-
-            for chair in doc.xpath('//h3[text()="Committee Chair"]/following-sibling::p'):
-                com.add_member(chair.text_content(), role='chairperson')
-                chairs.append(chair.text_content())
-
-            for member in doc.xpath('//h3[text()="Councilmembers"]/following-sibling::ul//a'):
-                if member.text_content() not in chairs:
-                    com.add_member(member.text_content(), role='member')
-
-            com.add_source(url)
-            self.save_committee(com)
+        comms = doc.xpath('//li[contains(@class,"node_committee-on")]/a')
```

Comment:
The ol' DC misspell causes this scraper rewrite to miss Health and Human Services, whose `class` is `"node_committe-on-health-and-human-services level_2"`.